discord/user.py

```diff
@@ -304,6 +310,15 @@ def display_name(self) -> str:
         if self.global_name:
             return self.global_name
         return self.name
+    
+    @property
+    def primary_guild(self) -> Optional[PrimaryGuild]:
+        """:class:`PrimaryGuild`: Returns the user's primary guild, if applicable.
+        
+        \nIf the user has not set a primary guild, ``None`` is returned."""
```

Comment:
it is clear that if no primary guild is set, itll return None
```suggestion
        """:class:`PrimaryGuild`: Returns the user's primary guild, if applicable."""
```